### PR TITLE
fix/cppcheck

### PR DESCRIPTION
**Pull request by ObiWahn**

### CODE REVIEW

File: utils/cppcheck.sh [hunk 1]
```diff
@@ -2,13 +2,14 @@
 
 ferr() { echo "$*"; exit 1; }
 
-if [[ -n $* ]];
+if [[ -n $* ]]; then
```
Comment (ObiWahn) [inline utils/cppcheck.sh]: :+1:

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7555/

http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-cppcheck/7/

**Comment by KVS85** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-ANY-cppcheck/12/

**Comment by KVS85** [comment]
http://jenkins.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7582/

**Comment by KVS85** [comment]
No cppcheck warnings or errors at all! Tests blue. oskar PR to be merged: https://github.com/arangodb/oskar/pull/206

**Pull request merged**